docker/Dockerfile

```diff
@@ -20,18 +24,13 @@ COPY cheribuild.json /root/.config/cheribuild.json
 
 # deps to build QEMU+elftoolchain:
 RUN apt-get update && apt-get install -y \
-  libtool pkg-config python-minimal autotools-dev automake autoconf libglib2.0-dev libpixman-1-dev \
+  libtool pkg-config python2-minimal autotools-dev automake autoconf libglib2.0-dev libpixman-1-dev \
```

Comment:
Is python2 actually needed?

Reply:
It used to be required for qemu but they migrated to py3 a while ago. I don't think it's required anymore but I could be wrong.

Reply:
I'll test removing  the python2 and push if that all "just works."